fMRIVolume/scripts/OneStepResampling.sh

```diff
@@ -284,18 +284,19 @@ then
     done
     
     #gdc warp space is input to input, affine is input to input, OutputTransform is input to MNI
-    xfmargs=(-warp "$GradientDistortionField.nii.gz" -fnirt "$InputfMRI"
+    xfmargs=(-warp "$GradientDistortionField".nii.gz -fnirt "$InputfMRI"
              -affine-series "$affseries" -flirt "$InputfMRI" "$InputfMRI"
-             -warp "$OutputTransform.nii.gz" -fnirt "$InputfMRI")
+             -warp "$OutputTransform".nii.gz -fnirt "$InputfMRI")
     
-    wb_command -volume-resample "$InputfMRI" "$WD/$T1wImageFile.$FinalfMRIResolution.nii.gz" CUBIC "$OutputfMRI.nii.gz" "${xfmargs[@]}" -nifti-output-datatype INT32
+    wb_command -volume-resample "$InputfMRI" "$WD/$T1wImageFile.$FinalfMRIResolution".nii.gz CUBIC "$OutputfMRI".nii.gz "${xfmargs[@]}" -nifti-output-datatype INT32
     
     #resample all-ones volume series with enclosing voxel to determine FOV coverage
     #yes, this is the entire length of the timeseries on purpose
-    fovcheck="$(mktemp --tmpdir OneStepResampleFovCheck_XXXXXX.nii.gz)"
-    tempfiles_add "$fovcheck"
+    fovcheck="$(tempfiles_create OneStepResampleFovCheck_XXXXXX.nii.gz)"
     wb_command -volume-math '1' "$fovcheck" -var x "$InputfMRI"
-    wb_command -volume-resample "$fovcheck" "$WD/$T1wImageFile.$FinalfMRIResolution.nii.gz" ENCLOSING_VOXEL "${OutputfMRI}_mask.nii.gz" "${xfmargs[@]}" -nifti-output-datatype INT32
+    #fsl's "nn" interpolation is just as pessimistic about the FoV as non-extrapolating interpolation
+    #so, in wb_command, use trilinear here instead of enclosing voxel to get the same FoV behavior as CUBIC
```

Comment:
Maybe add to the this comment: `(still not identical, but TRILINEAR closer than ENCLOSING_VOXEL)`